src/store/reducers/reducers.js

```diff
@@ -0,0 +1,43 @@
+import { DELETE_BOOK, CREATE_BOOK, UPDATE_BOOK } from "../actions/actions";
+
+const BOOKS_ID = "books";
+const newBooks = JSON.parse(localStorage.getItem(BOOKS_ID));
```

Comment:
Почему у тебя в файле две одинаковые переменные, странно что книги с localStorage называются новыми
```suggestion
const booksFromStorage = JSON.parse(localStorage.getItem(BOOKS_ID));
```